lib/developer_portal/lib/liquid/tags/active_docs.rb

```diff
@@ -16,19 +16,21 @@ def initialize(tag_name, params, tokens)
         super
         if params =~ Syntax
           @version  = $1.strip
-          @services = $2.nil? ? [] : $2.split(",").map(&:strip)
+          @system_names = $2.nil? ? [] : $2.split(",").map(&:strip)
         end
       end
 
+      attr_reader :system_names
```

Comment:
This will be public? So they can use it in the liquid template?

Reply:
I didn't think this through. We can remove that.

Reply:
Done